src/subsidy.cpp

```diff
@@ -41,7 +41,7 @@ void Subsidy::AwardTo(CompanyID company)
 	assert(!this->IsAwarded());
 
 	this->awarded = company;
-	this->remaining = SUBSIDY_CONTRACT_MONTHS;
+	this->remaining = _settings_game.difficulty.subsidy_duration * 12;
```

Comment:
Is there a NUM_MONTHS constant somewhere that could be used?

Reply:
I made one; let me know if it's in the wrong place.